project2/style.css

```
@@ -0,0 +1,66 @@
+.title {
+    font-family: 'ZCOOL QingKe HuangYou', cursive;
```

This font itself is fine, but the fallback browser defaults for "cursive" fonts are pretty terrible. Even if Google Fonts recommends it, I suggest changing this to either "serif" or "sans-serif," whichever is close to the font you're using:
```suggestion
    font-family: 'ZCOOL QingKe HuangYou', sans-serif;
```